zoo-css-challenge/style.css

```
@@ -85,45 +90,71 @@ footer {
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  /* bottom: 20px; */
   right: 20px;
-
+  top: 20px;
   display: flex;
   justify-content: center;
   align-content: center;
   flex-direction: column;
+  transition: all 0.25s ease-in;
 }
 
 #badge:hover {
   background-color: yellow;
+  transition: all 0.25s ease-in;
 }
 
 #badge p {
   text-align: center;
   font-size: 125%;
 }
 
+/* .image-container{
+  display: flex;
+justify-content: space-evenly;
+} */
+
 .image-container img {
   vertical-align: top;
-  border: 5px solid #747c92;
+  /* border: 5px solid #747c92; */
 }
 
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  border: 5px solid #747c92;
+}
+
+.image-container img:nth-child(2){
+  border: 5px solid #88ac07;
+}
+
+.image-container img:last-child{
+  border: 5px solid #07ac10;
 }
 
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
 
+.tigers h2, h3{
+  color: #FFFFFF;
+}
 .giraffe {
-  background-color: #465775;
+  background-color: #483C46;
   color: #56e39f;
 }
 
-.giraffe h2 {
-  color: #ef6f6c;
+.giraffe h2, h3 {
```

```suggestion
.giraffe h2, .giraffe h3 {
```
The second selector here targets all `h3` elements. The task is to only target the `h3`s in the giraffe section, so we should change it to `.giraffe h3` accordingly